meta-lmp-base/recipes-core/initrdscripts/initramfs-framework/cryptfs_pkcs11

```diff
@@ -5,6 +5,13 @@ cryptfs_check_pkcs11() {
 	[ ! -c /dev/tee0 ] && fatal "Linux OP-TEE device node not found"
 }
 
+cryptfs_check_token_pkcs11() {
+	MODULE=/usr/lib/libckteec.so.0
```

Comment:
Module is also defined at cryptfs_enroll_pkcs11, better to move to global to avoid duplication.